vocabularies/voc4cat/0007106.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007106>
```

Comment:
The concept "continuous stirred tank reactor" is already defined: https://w3id.org/nfdi4cat/voc4cat_0007019 - You may add "continuously stirred tank reactor" as alternate label but I think it is not required since stemming will make continuous/continuously the same.

Reply:
sorry, I missed both definitions due to slight spelling changes. 
Responding to the:
- as used in modeling
- as found in reality
we should introduce two concepts and also separate them at the highest hierarchical level so that these concepts will not be misused. As such I hope some of you can recommend how we might separate then. 

My recommendation would be:
- realized reactor type
- modeled reactor type
alternatively:
- reactor design type
- reactor model type

I would appreciate further ideas.

Reply:
I like 
* reactor design type (which implies we should change the pref. label concept of https://w3id.org/nfdi4cat/voc4cat_0007018 to this)
* reactor model type
best. Do you want to add them here or in a separate PR? To finalize the review of your defs here I only need to know which type you want to define here.

Reply:
@dalito: how do i recommit a file like the changed [voc4cat.xlsx](https://github.com/nfdi4cat/voc4cat/files/15160852/voc4cat.xlsx) correctly into this pull request? Sry due to the CI Pipeline modifying the original excel file i am a bit lost.

Reply:
How do you use git? From command line?

Assuming you checked out your fork, are in the root of the repository and have activated the branch where you created the PR from:

* To update your local repo: `git pull`
* Re-add updated voc4cat.xls: `git add pull`
* Commit changes: `git commit -m "my change notes"`

Yesterday, I extended the notes on how to work with git in the readme.